### ISSUE TITLE
make wait_true() back-compatiable

### ISSUE THREAD

**Issue by djwgit** [appium/ruby_lib#778]
## This is a
* [x] Bug report
* [ ] Question
* [ ] Freature Request

## Summary
wait_true() used to be able to take a number as a time-out value, e.g. wait_true(10){ }
in the implementation, it checks if the value passed in is a number, it converts it into a hash with :timeout value.    

```
      # @private
      def process_wait_opts(opts)
        opts = { timeout: opts } if opts.is_a?(Numeric)
        raise 'opts must be a hash' unless opts.is_a? Hash
        opts
      end
```

this makes it convenient to write as
```
wait_true(10){ ... }
instead of 
wait_true(timeout:3) { ... }
```
while, with the change made on Oct 11, 2017, the function only takes a hash, it is not back-compatible.
nice to have it back-compatible.
otherwise, it is a trouble for whoever has been using wait_true(timeout)  :-(

in the change made here
https://github.com/appium/ruby_lib/commit/798c3aae8f5ec42f0286b17624e2d667e362b24c#diff-2cfefaf9bee8b4cb8efe5d2d5f42297f

## Environment
* Appium version (or git revision):
* `ruby_lib` version: 9.12.0
* Mobile platform/version/device under test:

## Link to Appium/Ruby logs

Create a [GIST](https://gist.github.com) which is a paste of your _full_ Appium logs, and link them here.

**Comment by KazuCocoa** [comment]
Thanks.
Sorry that was my missing...

**Comment by KazuCocoa** [comment]
https://github.com/appium/ruby_lib/blob/master/CHANGELOG.md#v9121
Could you check the latest one? @djwgit

**Comment by djwgit** [comment]
thanks @KazuCocoa  fixed in v9.12.1
